starport/pkg/yaml/yaml.go

```diff
@@ -0,0 +1,38 @@
+package yaml
+
+import (
+	"context"
+	"errors"
+	"strings"
+
+	"github.com/goccy/go-yaml"
+	"github.com/goccy/go-yaml/parser"
+)
+
+func ParseString(ctx context.Context, obj interface{}, paths ...string) (string, error) {
```

Comment:
Can we add a description to this func about what does it do exactly with given obj and paths, and what's the difference between a yaml marshaler pkg directly or using this func?